app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -62,7 +63,8 @@ class MainFragment : Fragment() {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults)
         when(requestCode){
             CAMERA_PERMISSION_REQUEST_CODE -> {
-                if(grantResults.isNotEmpty() && grantResults[0] == PackageManager.PERMISSION_GRANTED){
+                //Checks if array has the correct code
```

Comment:
I also changed this conditional to not have two comparisons and rather just have one.